test/HotPotato.Core.Test/Http/Default/HttpRequestTest.cs

```diff
@@ -26,6 +28,22 @@ public void HttpRequest_DecodedPath_DecodesSpecialCharactersWithMethodCtor()
 			Assert.Equal(ExpectedPath, result);
 		}
 
+		[Fact]
+		public void HttpRequest_DecodedPath_DecodesAndReturnsNonNullRelativePathMethodCtor()
```

Comment:
I finally looked at Rich's Bitbucket changes a couple months ago where he added a bunch of changes to the Hot Potato Request constructor to take in relative paths. I wish I could've told him that he could just pass UriKind.Relative into the Uri ctor and avoid his rewrites. @lambchop4prez

Reply:
Is that something that we may want to put into documentation somewhere?